cypher-shell/src/main/java/org/neo4j/shell/cli/InteractiveShellRunner.java

```diff
@@ -28,36 +31,42 @@
  */
 public class InteractiveShellRunner implements ShellRunner, SignalHandler {
     static final String INTERRUPT_SIGNAL = "INT";
-    private final static AnsiFormattedText freshPrompt = AnsiFormattedText.s().bold().append("neo4j> ");
+    private final static String freshPrompt = "> ";
```

Comment:
so we change this because the database name should appear here?

Reply:
Yes